app/services/tasks.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any
+from flask import jsonify
+
+from app import db
+from app.models.models import Tasks
+
+
+def create_task(model: Any, id: int, title: str, description: str):
+    task = model(
+        id = id,
+        title = title,
+        description = description
+    )
+    db.session.add(task)
+    db.session.commit()
+
+    return task
+
+def get_task_by_id(id):
+    task = Tasks.query.get(id)
+    return task
+
+def update_task(data ,id):
+    task = get_task_by_id(id)
+    if "id" in data:
+        return jsonify(message="You cannot update the id")
```

Comment:
It doesn't matter, id is there or not. If you don't update through code, It won't be updated.
In mongodb, record creates whichever field you pass, Not the same case with SQL.